packages/picasso-lab/src/DrilldownMenuItem/test.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { useCallback, useMemo, useState } from 'react'
+import { fireEvent, render, TestingPicasso } from '@toptal/picasso/test-utils'
+
+import DrilldownMenuItem from './DrilldownMenuItem'
+import DrilldownMenuContext, {
+  DrilldownMenuContextProps
+} from '../DrilldownMenu/DrilldownMenuContext'
+
+const TestDrilldownMenuItem = () => {
```

Comment:
It does not render the item; it renders the menu. What do you think about such a change?

```suggestion
// Testing item separately does not make sense. Rendering inside the menu component.
const TestDrilldownMenu = () => {
```